lib/questions.js

```diff
@@ -13,6 +13,7 @@ const choices = (config) => {
     choicesList.push({
       value: emoji + configType,
       name: `${chalk.bold(configType)} ${description}`,
+      key: type.argKeys || [],
```

Comment:
What if the user does not specify a Array in the `.sgcrc`, e.g. a string? This would be passed... Does this still work?  

I do not think so: see line 51

```js
const argChoice = choicesList.find(choice => choice.key.includes(modifiedArgv.t));
```

propably a check like: 

```

Array.isArray(type) && type.length > 0 ? type : [],
```

but also we should have a error message for the user. Otherwise the specified type as string is simple skipped and the user does not understand why.

Reply:
True, I am gonna add a typechecker here :)